source/channel_bootstrap.c

```diff
@@ -183,7 +157,7 @@ int aws_client_bootstrap_set_alpn_callback(
     return AWS_OP_SUCCESS;
 }
 
-void aws_client_bootstrap_destroy(struct aws_client_bootstrap *bootstrap) {
+void aws_client_bootstrap_release(struct aws_client_bootstrap *bootstrap) {
```

Comment:
Does aws_server_bootstrap need this same treatment? I know it doesn't have a resolver right now, but eventually a server might want to resolve peers? Or would we delegate that to connection handlers?

Reply:
The only time you'd need to resolve peers (say, to connect to something else), you'd go through a client bootstrap. Can you think of a different use-case?

Reply:
I did change the server to use release instead of destroy